contrib/container/docker-compose.yml

```diff
@@ -83,7 +83,7 @@ services:
         # If you wish to specify a particular InvenTree version, do so here
         image: inventree/inventree:${INVENTREE_TAG:-stable}
         container_name: inventree-worker
-        command: invoke worker
+        command: invoke dev.worker
```

Comment:
Is that really a dev command here?

Reply:
I would consider it so - but maybe it is more of an int? I do not really see endusers running it

Reply:
Endusers are not expected to type this into their CLI - or can you point me to a scenario in which a user should manually start a worker up?

internal. things that are needed for correct operation of the system or CI but are not meant for regular usage. At some point we can consider hiding those by default to not overwhelm the standard administrator.

Reply:
ok, that makes sense now. I think then this would be more of an int task, as this is used to run a prod setup, but it's not directly used by a system administrator.